docs/reference/persister.rst

```diff
@@ -57,3 +57,29 @@ Currently we support the following, although we highly recommend you contribute
 
 Note that the :py:class:`LocalTrackingClient <burr.tracking.client.LocalTrackingClient>` leverages the :py:class:`BaseStateLoader <burr.core.persistence.BaseStateLoader>` to allow loading state,
 although it uses different mechanisms to save state (as it tracks more than just state).
+
+
+
+MongoDB Persister
+=================
+
+The `MongoDBPersister` class is used to save and load state data to/from a MongoDB database.
+
+Example usage:
+
+.. code-block:: python
+
+    from burr.integrations.persisters.b_mongodb import MongoDBPersister
+    from burr.core import state
+
+    persister = MongoDBPersister(uri='mongodb://user:pass@localhost:27017', db_name='mydatabase', collection_name='mystates')
+    persister.save(
+        partition_key='example_partition',
+        app_id='example_app',
+        sequence_id=1,
+        position='example_position',
+        state=state.State({'key': 'value'}),
+        status='completed'
+    )
+    loaded_state = persister.load(partition_key='example_partition', app_id='example_app', sequence_id=1)
+    print(loaded_state)
```

Comment:
Actually all that was needed was just to do something like this:

```
.. autoclass:: burr.integrations.persisters.b_mongodb.MongoDBPersister
   :members:
   .. automethod:: __init__
```

What you have here should show up in the documentation string in the class, that will then be surfaced there. So can you please remove this, and replace if with what I just put above, but placing it just after the Redis one please.